packages/kit/types/internal.d.ts

```diff
@@ -157,15 +157,6 @@ export type RecursiveRequired<T> = {
 
 export type RequiredResolveOptions = Required<ResolveOptions>;
 
-export interface Respond {
```

Comment:
inlined this into the function because it was annoying that "go to definition" showed two positions (the type and the implementation) when I just want to go to the implementation